src/actions/guides/database/models.cr

```diff
@@ -177,12 +177,12 @@ class Guides::Database::Models < GuideAction
     * `Int64` - `bigint` column type.
     * `Float64` - `numeric` column type.
     * `Bool` - `boolean` column type.
-    * `Time` - `timestamp with time zone` (`timestampz`) column type.
+    * `Time` - timestamp with time zone (`timestamptz`) column type.
```

Comment:
```suggestion
    * `Time` - `timestamp with time zone` (`timestamptz`) column type.
```

The backticks here are actually correct. It's what's shown if you did `\d comments` in a `psql`.

Reply:
TIL. Thanks!